libraries/__shared__/karma-plugins/karma-custom-html-reporter/index.js

```diff
@@ -120,6 +120,9 @@ var HTMLReporter = function(baseReporterDecorator, config, emitter, logger, help
       overview = suite.ele('header', { class: 'overview' });
 
       // Assemble the Overview
+      let pkg = require(path.join(process.cwd(), './package.json'));
+      let libraryVersion = pkg.dependencies[process.env.LIBRARY_NAME];
+      overview.ele('div', { class: 'title is-6' }, 'Version: ' + libraryVersion);
```

Comment:
Same approach as the json reporter: https://github.com/webcomponents/custom-elements-everywhere/blob/11fe3ab680af3b3fb33a864e95d012f032761de2/libraries/__shared__/karma-plugins/karma-custom-json-reporter/index.js#L32-L34